### DISCUSSION TITLE
second vacuum map card

### DISCUSSION THREAD

**Discussion by BenjiMcKenzie** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#563] [Q&A]
Hi all,

one question to all, i integrated my second vacuum to HA, i can controll it, but i cannot get the map from the second vacuum.
If i add the map extractor card for the second vacuum, it only shown the map from my first vacuum.

Got anybody a solution for my problem?

greetings
Benji

**Comment by PiotrMachowski** [comment]
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/discussions/108
